phpstan.neon

```diff
@@ -1,71 +1,16 @@
 parameters:
-    bootstrap: %rootDir%/../../../phpstan-bootstrap.php
-
+    checkGenericClassInNonGenericObjectType: false
+    checkMissingIterableValueType: false
     ignoreErrors:
-    - '#Call to an undefined method .+LocatorLocatorInterface::.+\(\).#'
-
-services:
-    -
-        class: PhpStan\DynamicType\FacadeDynamicTypeExtension
-        tags:
-            - phpstan.broker.dynamicMethodReturnTypeExtension
-
-    -
-        class: PhpStan\DynamicType\QueryContainerDynamicTypeExtension
-        tags:
-            - phpstan.broker.dynamicMethodReturnTypeExtension
-
-    -
-        class: PhpStan\DynamicType\ConsoleDynamicTypeExtension
-        tags:
-            - phpstan.broker.dynamicMethodReturnTypeExtension
-
-    -
-        class: PhpStan\DynamicType\ControllerDynamicTypeExtension
-        tags:
-            - phpstan.broker.dynamicMethodReturnTypeExtension
-
-    -
-        class: PhpStan\DynamicType\PluginDynamicTypeExtension
-        tags:
-            - phpstan.broker.dynamicMethodReturnTypeExtension
-
-    -
-        class: PhpStan\DynamicType\ServiceDynamicTypeExtension
-        tags:
-            - phpstan.broker.dynamicMethodReturnTypeExtension
-
-    -
-        class: PhpStan\DynamicType\ConfigDynamicTypeExtension
-        tags:
-            - phpstan.broker.dynamicMethodReturnTypeExtension
-
-    -
-        class: PhpStan\DynamicType\ClientDynamicTypeExtension
-        tags:
-            - phpstan.broker.dynamicMethodReturnTypeExtension
-
-    -
-        class: PhpStan\DynamicType\ClientPluginDynamicTypeExtension
-        tags:
-            - phpstan.broker.dynamicMethodReturnTypeExtension
-
-    -
-        class: PhpStan\DynamicType\YvesFormTypeDynamicTypeExtension
-        tags:
-            - phpstan.broker.dynamicMethodReturnTypeExtension
-
-    -
-        class: PhpStan\DynamicType\ZedFormTypeDynamicTypeExtension
-        tags:
-            - phpstan.broker.dynamicMethodReturnTypeExtension
-
-    -
-        class: PhpStan\DynamicType\YvesFactoryDynamicTypeExtension
-        tags:
-            - phpstan.broker.dynamicMethodReturnTypeExtension
-
-    -
-        class: PhpStan\DynamicType\ZedFactoryDynamicTypeExtension
-        tags:
-            - phpstan.broker.dynamicMethodReturnTypeExtension
+        - '#Call to method .+\(\) on an unknown class Generated\\Zed\\Ide\\AutoCompletion#'
+        - '#Call to static method create\(\) on an unknown class .+Query.#'
+        - '#Call to method .+\(\) on an unknown class Orm\\.+#'
+        - '#Return typehint of method .+ has invalid type Orm\\.+#'
+        - '#Parameter \$.+ of method .+ has invalid typehint type Orm\\.+#'
+        - '#Call to method.+on an unknown class.+AutoCompletion#'
+        - '#.+Transfer.+#'
```

Comment:
That must be removed
Instead use https://spryker.atlassian.net/wiki/spaces/CORE/pages/2868248862/Standalone+module+testing to make transfers available and allow correct PHPStan run

Reply:
Updated